doc/voq/voq_hld.md

```diff
@@ -477,7 +477,99 @@ The system port configuration has the parameters listed below.
+
+The show commands are under **show chassis** commands group as shown below
+
+**Help info of show chassis command group**
+```
+admin@Linecard2:~$ show chassis -h
+Usage: show chassis [OPTIONS] COMMAND [ARGS]...
+
+  Chassis commands group
+
+Options:
+  -?, -h, --help  Show this message and exit.
+
+Commands:
+  modules           Show chassis-modules information
+  system-neighbors  Show VOQ system neighbors information
+  system-ports      Show VOQ system ports information
+admin@Linecard2:~$ 
+```
+### 2.7.1 System ports show command
+The information for the system ports are taken from the **SYSTEM_PORT_TABLE** of **APPL_DB** from a specific namespace 
+#### Syntax
+```
+admin@Linecard2:~$ show chassis system-ports -h
+Usage: show chassis system-ports [OPTIONS] [SYSTEMPORTNAME]
+
+  Show VOQ system ports information
+
+Options:
+  -d, --display [all]  Show internal interfaces  [default: all]
+  -n, --namespace []   Namespace name or all
+  --verbose            Enable verbose output
+  -h, -?, --help       Show this message and exit.
+admin@Linecard2:~$
+```
+#### Sample output
+```
+admin@Linecard2:~$ show chassis system-ports
+            System Port Name    Port Id    Switch Id    Core    Core Port    Speed
+----------------------------  ---------  -----------  ------  -----------  -------
+       Linecard2|Asic0|Asic0        192            6       0            0      10G
+   Linecard2|Asic0|Ethernet0        193            6       0            1     100G
+   Linecard2|Asic0|Ethernet1        194            6       0            2     100G
+   Linecard2|Asic0|Ethernet2        195            6       0            3     100G
+Linecard2|Asic0|Ethernet-IB0        229            6       1           37      10G
+.
+.
+.
+       Linecard4|Asic0|Asic0        576           18       0            0      10G
+   Linecard4|Asic0|Ethernet0        577           18       0            1     100G
+   Linecard4|Asic0|Ethernet1        578           18       0            2     100G
+   Linecard4|Asic0|Ethernet2        579           18       0            3     100G
+Linecard4|Asic0|Ethernet-IB0        613           18       1           37      10G
+.
+.
+.
+```
+**Note:**
+
+   - In multi asic voq systems, for system ports since the configurations in all asics are required to be identical, the output of show for system ports in all namespaces will show multiple entries of the same system port info. The number of entries will be equal to the number of namespaces in the host.
+
+### 2.7.2 System neighbors show command
+The information for the system neighbors are taken from the **SYSTEM_NEIGH_TABLE** of **CHASSIS_APP_DB** in the supervisor card. 
+#### Syntax
+```
+admin@Linecard2:~$ show chassis system-neighbors -h
+Usage: show chassis system-neighbors [OPTIONS]
+
+  Show VOQ system neighbors information
+
+Options:
+  -d, --display [all]  Show internal interfaces  [default: all]
```

Comment:
Same as above. Do we need the `-d` option ?

Reply:
No. We do not need this. I'll remove this.

Reply:
Fixed. Removed "-d" option.